lib/package-keymap-view.coffee

```diff
@@ -8,21 +9,53 @@ class PackageKeymapView extends View
   @content: ->
     @section class: 'section', =>
       @div class: 'section-heading icon icon-keyboard', 'Keybindings'
+      @div class: 'checkbox', =>
+        @label for: 'toggleKeybindings', =>
+          @input id: 'toggleKeybindings', type: 'checkbox', outlet: 'keybindingToggle'
+          @div class: 'setting-title', 'Use Keybindings?'
```

Comment:
What do you think about dropping the `?` here? I think we tend to avoid punctuation when rendering setting names.